project/threads/templates/threads/static_templates/landing.html

```diff
@@ -114,15 +111,13 @@ <h5 class="content-subtitle">Personalized Policy Feed</h5>
                         <i class="large material-icons">location_city</i>
                     </div>
                     <div class="col s12 m6">
-                        <h5 class="content-subtitle">Citizen/Representative Engagement</h5>
+                        <h5 class="content-subtitle">{% trans 'Citizen/Representative Engagement' %} </h5>
                         <p class="content-text">
-                            CiviWiki's core goal is to engage citizens and their representatives, with the goal of making government more accountable.
+                            {% trans "CiviWiki's core goal is to engage citizens and their representatives, with the goal of making government more accountable. "%}
                             <br>
-                            <br> First, CiviWiki will organize user's policy profile and compare it to every political candidate in the user's district. This quick, detailed, comparison will help users make informed votes, and we believe increased voter
-                            confidence will increase voter turnout.
+                            <br> {% trans "First, CiviWiki will organize user's policy profile and compare it to every political candidate in the user's district. This quick, detailed, comparison will help users make informed votes, and we believe increased voter confidence will increase voter turnout. " %}
                             <br>
-                            <br> Second, CiviWiki will collect anonymized user data and forward district level statistics to representatives. With a critical mass of users, we believe timely district level polling data will influence representatives'
-                            votes
+                            <br> {% trans "Second, CiviWiki will collect anonymized user data and forward district level statistics to representatives. With a critical mass of users, we believe timely district level polling data will influence representatives votes " %}
```

Comment:
Trailing whitespace, grammar

```suggestion
                            <br> {% trans "Second, CiviWiki will collect anonymized user data and forward district-level statistics to representatives. With a critical mass of users, we believe timely district level polling data will influence representatives votes." %}
```